maven checkstyle failed

The issue tracker is **ONLY** used for bug report(feature request need to follow [RIP process](https://github.com/apache/rocketmq/wiki/RocketMQ-Improvement-Proposal)). Keep in mind, please check whether there is an existing same report before your raise a new one.

Alternately (especially if your communication is not a bug report), you can send mail to our [mailing lists](http://rocketmq.apache.org/about/contact/). We welcome any friendly suggestions, bug fixes, collaboration and other improvements.

Please ensure that your bug report is clear and that it is complete. Otherwise, we may be unable to understand it or to reproduce it, either of which would prevent us from fixing the bug. We strongly recommend the report(bug report or feature request) could include some hints as the following:

**BUG REPORT**

1. Please describe the issue you observed:

- What did you do (The steps to reproduce)?
mvn clean install
- What did you expect to see?
build successfully
- What did you see instead?
checkstyle failed
> [INFO] 
[INFO] --- maven-checkstyle-plugin:2.17:check (verify) @ rocketmq-common ---
[INFO] Starting audit...
/Users/chenrl/Desktop/openSource/rocketmq/common/src/main/java/org/apache/rocketmq/common/utils/NameServerAddressUtils.java:37:60: error: '+' is not preceded with whitespace.
/Users/chenrl/Desktop/openSource/rocketmq/common/src/main/java/org/apache/rocketmq/common/utils/NameServerAddressUtils.java:37:61: error: '+' is not followed by whitespace.
Audit done.
2. Please tell us about your environment:

3. Other information (e.g. detailed explanation, logs, related issues, suggestions how to fix, etc):

**FEATURE REQUEST**

1. Please describe the feature you are requesting.

2. Provide any additional detail on your proposed use case for this feature.

2. Indicate the importance of this issue to you (blocker, must-have, should-have, nice-to-have). Are you currently using any workarounds to address this issue?

4. If there are some sub-tasks using -[] for each subtask and create a corresponding issue to map to the sub task:

- [sub-task1-issue-number](example_sub_issue1_link_here): sub-task1 description here, 
- [sub-task2-issue-number](example_sub_issue2_link_here): sub-task2 description here,
- ...

--- a/client/src/main/java/org/apache/rocketmq/client/ClientConfig.java
+++ b/client/src/main/java/org/apache/rocketmq/client/ClientConfig.java
@@ -94,7 +94,7 @@ public class ClientConfig {
 
     public void changeInstanceNameToPID() {
         if (this.instanceName.equals("DEFAULT")) {
-            this.instanceName = UtilAll.getPid()+ "#" + System.nanoTime();
+            this.instanceName = UtilAll.getPid() + "#" + System.nanoTime();
         }
     }
 

--- a/common/src/main/java/org/apache/rocketmq/common/utils/NameServerAddressUtils.java
+++ b/common/src/main/java/org/apache/rocketmq/common/utils/NameServerAddressUtils.java
@@ -34,7 +34,7 @@ public class NameServerAddressUtils {
         if (StringUtils.isEmpty(endpoint)) {
             return null;
         }
-        return endpoint.substring(endpoint.lastIndexOf("/")+1, endpoint.indexOf('.'));
+        return endpoint.substring(endpoint.lastIndexOf("/") + 1, endpoint.indexOf('.'));
     }
 
     public static String getNameSrvAddrFromNamesrvEndpoint(String nameSrvEndpoint) {

--- a/example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java
+++ b/example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java
@@ -16,18 +16,18 @@
  */
 package org.apache.rocketmq.example.ordermessage;
 
-import java.io.UnsupportedEncodingException;
-import java.util.List;
 import org.apache.rocketmq.client.exception.MQBrokerException;
 import org.apache.rocketmq.client.exception.MQClientException;
 import org.apache.rocketmq.client.producer.DefaultMQProducer;
-import org.apache.rocketmq.client.producer.MQProducer;
 import org.apache.rocketmq.client.producer.MessageQueueSelector;
 import org.apache.rocketmq.client.producer.SendResult;
 import org.apache.rocketmq.common.message.Message;
 import org.apache.rocketmq.common.message.MessageQueue;
 import org.apache.rocketmq.remoting.common.RemotingHelper;
 import org.apache.rocketmq.remoting.exception.RemotingException;
+
+import java.io.UnsupportedEncodingException;
+import java.util.List;
 
 public class Producer {
     public static void main(String[] args) throws UnsupportedEncodingException {

